### PR TITLE
Add a qbs project

### PR DESCRIPTION
**Pull request by orgads**

### CODE REVIEW

File: dwarfstack.qbs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    cpp.dynamicLibraries: [
+        "dbghelp",
+        "gdi32",
+        "stdc++",
+    ]
+    cpp.includePaths: [
+        "include",
+        "mgwhelp",
+        "libdwarf",
+    ]
```
Comment (ssbssa) [inline dwarfstack.qbs]: Did it work for you without adding "zlib", here?
I guess you already have a zlib.h in your system which gcc can use.

Comment (orgads) [reply dwarfstack.qbs]: Good point :)

File: dwarfstack.qbs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        name: "libdwarf"
+        prefix: "libdwarf/"
+        files: [
+            "dwarf_*.c",
+            "pro_encode_nm.c",
+        ]
+        excludeFiles: [
+            "dwarf_errmsg_list.c",
+            "dwarf_line_table_reader_common.c",
+        ]
```
Comment (ssbssa) [inline dwarfstack.qbs]: One other thing. I needed this define here as well to fix a few warnings:
cpp.defines: "DW_TSHASHTYPE=uintptr_t"

This was very easy to miss in my Makefile.

Comment (orgads) [reply dwarfstack.qbs]: Done

**Pull request merged**